go/httpapi-eventbridge/README.md

```diff
@@ -0,0 +1,29 @@
+# Amazon API Gateway HTTP API to Amazon EventBridge
+
+This pattern creates an HTTP API endpoint that directly integrates with Amazon EventBridge
+
+## How it works
+
+This pattern creates an Amazon API gateway HTTP API endpoint. The endpoint uses service integrations to directly connect to Amazon EventBridge. An EventBridge rule sends all events to Cloudwatch Logs.
+
+## Deploy
+
+Run `cdk deploy`. This will deploy / redeploy your Stack to your AWS Account.
```

Comment:
Explain here how to get the endpoint of the stack.

Reply:
Added a hint that the endpoint is logged during `cdk deploy` ✅